cmd/legacy/subcmd.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+package legacy
```

Comment:
IMO legacy is a bit abstract, and when legacy is the only option it can be confusing. Can we refer to this as a specific control plane type? i.e. teams vs teams(new)